redash/query_runner/snowflake.py

```diff
@@ -43,6 +45,9 @@ def configuration_schema(cls):
                 "account": {"type": "string"},
                 "user": {"type": "string"},
                 "password": {"type": "string"},
+                "private_key_File": {"type": "string"},
+                "private_key_b64": {"type": "string"},
```

Comment:
I think we don't need private_key_b64 as we have private_key_File.

Reply:
i don't agree, but i'm tired of discussing this.